app/knights/generate_props_for_knight.py

```diff
@@ -0,0 +1,46 @@
+from typing import Callable, Any
+
+from app.knights.knight_constructor import KnightFighter
+
+
+def armour(func: Callable) -> Any:
+    def wrapper(item: KnightFighter) -> Any:
```

Comment:
```suggestion
    def wrapper(knight: KnightFighter) -> Any:
```
може так буде трохи зрозуміліше